Adding paths menu option and max paths

mwfarb opened the pull request:
This PR adds a new sub-menu to the Paths tab to consolidate Paths-only option controls.
- Added Paths menu.
- Moves Destination AS input box to new menu.
- Adds Max Paths input box to new menu.
- Closes #31.

@ercanucan, @mlegner Please have a look.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/netsec-ethz/scion-viz/39)
<!-- Reviewable:end -->

mlegner commented:
This works well for me. However, it feels a little counterintuitive that the "submit" button is above some of the input fields.
Also, the placeholder for the Destination AS (2-1) could be replaced with one that actually exists (1-1 or 2-21).

---

Review status: 0 of 4 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-:-Kyq7GhNka23nO8jcYaS:b-s31xz)*
<!-- Sent from Reviewable.io -->

mwfarb commented:
@mlegner I changed the 'Submit' button to 'Request Data' since it's available for every tab. I also added a 'Update Paths' button at the end of the Paths inputs so users will intuit that their granular Paths menu changes will used. Placeholder updated as suggested.

@ercanucan Comments addressed.

---

Review status: 0 of 4 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

*[python/as_viewer.py, line 29 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-KyprEsZNPGBeEvOuVYM:-Kz0g1370TsGIhS5MG1K:b-896fix) ([raw file](https://github.com/netsec-ethz/scion-viz/blob/dc8cc0ebb89ba6c7f65dd8660e04bbf323edfb2a/python/as_viewer.py#L29)):*
<details><summary><i>Previously, ercanucan (ercanucan) wrote…</i></summary><blockquote>

please format the imports according the lint guidelines.
</blockquote></details>

Done.

---

*[python/web/asviz/views.py, line 26 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-KyprO6MyoxGtlgaH6p7:-Kz0g3LzIFIvCHdqFJMu:b-896fix) ([raw file](https://github.com/netsec-ethz/scion-viz/blob/dc8cc0ebb89ba6c7f65dd8660e04bbf323edfb2a/python/web/asviz/views.py#L26)):*
<details><summary><i>Previously, ercanucan (ercanucan) wrote…</i></summary><blockquote>

ditto regarding imports.
</blockquote></details>

Done.

---

*[python/web/asviz/views.py, line 825 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-KyprxowxR3R4OmrhZIY:-Kz5VxsOgNmdiCRvwmGS:b-6t9wdv) ([raw file](https://github.com/netsec-ethz/scion-viz/blob/dc8cc0ebb89ba6c7f65dd8660e04bbf323edfb2a/python/web/asviz/views.py#L825)):*
<details><summary><i>Previously, ercanucan (ercanucan) wrote…</i></summary><blockquote>

formatting here looks inconsistent with too many fields on this line.
</blockquote></details>

Changed dictionary access to remove redundacies.

---

*[python/web/asviz/views.py, line 843 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-KypryoLkn-xmW3WiApl:-Kz5W1iORgwH9kBfYOUD:b-whxqss) ([raw file](https://github.com/netsec-ethz/scion-viz/blob/dc8cc0ebb89ba6c7f65dd8660e04bbf323edfb2a/python/web/asviz/views.py#L843)):*
<details><summary><i>Previously, ercanucan (ercanucan) wrote…</i></summary><blockquote>

Also formatting here looks inconsistent with too many fields on this line. The two functions seem to contain a lot of duplicate code. Maybe the data contained here could reside in a separate Class?
</blockquote></details>

I've converted this dictionary to be used more directly.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-:-Kz4Swv4scd0aUi21kEJ:b-8wfse5)*
<!-- Sent from Reviewable.io -->

mlegner commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 4 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-viz/39#-:-Kz7q22IrCL3VxYJ8K4n:bnfp4nl)*
<!-- Sent from Reviewable.io -->
